llm/sglang/README.md

```diff
@@ -28,34 +28,115 @@ service:
   replicas: 2
 ```
 
-The entire Service YAML can be found here: [sglang.yaml](sglang.yaml).
+The entire Service YAML can be found here: [llava.yaml](llava.yaml).
 
 2. Start serving by using [SkyServe](https://skypilot.readthedocs.io/en/latest/serving/sky-serve.html) CLI:
 ```bash
-sky serve up -n sglang sglang.yaml
+sky serve up -n sglang-llava llava.yaml
```

Comment:
We now support `--env`, it would be easier to let the user specify it in the command arg. :)
```suggestion
sky serve up -n sglang-llava llava.yaml --env HF_TOKEN=<your-huggingface-token>
```